aws/main.tf

```diff
@@ -87,17 +96,16 @@ terraform {
   }
 }
 
-variable "region" {
+variable "subnet_id" {
+  default = ""
   type    = string
-  default = "us-west-2"
 }
 
-provider "aws" {
-  region = var.region
+variable "aws_region" {
+  type    = string
+  default = "us-west-2"
 }
 
-variable "availability_zone" {
-  description = "The AWS AZs to deploy the infrastructure on"
-  default     = ["us-west-2a"]
-  type        = list(string)
+provider "aws" {
+  region = var.aws_region
 }
```

Comment:
Why delete the availability_zone variable?

Reply:
fixed, not sure how that got removed